contributing/contributing_datasets.md

```diff
@@ -0,0 +1,287 @@
+---
+layout: default
+title: Contributing datasets
```

Comment:
Does the d need to be capitalized? Not sure if it shows up in the docsite rendering.

Reply:
Per https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings, titles are supposed to be in sentence case.